src/mono/wasm/runtime/binding_support.js

```diff
@@ -142,6 +142,9 @@ var BindingSupportLib = {
 		},
 
 		js_string_to_mono_string: function (string) {
+			if (string === null || typeof string === "undefined")
+				return null;
```

Comment:
This should maybe return `0` here, since IIUC, this function returns an address.

Just reading the code:
- this gets called by `js_to_mono_obj`
- `js_to_mono_enum` would pass the result of that to `mono_wasm_new_root`, which would throw because it's not a number
```suggestion
				return 0;
```

Reply:
better safe than sorry.  Let's get all the tests stood up.